NEWS

```diff
@@ -3,6 +3,14 @@ For full details, see the git log at: https://github.com/ksh93/ksh
 
 Any uppercase BUG_* names are modernish shell bug IDs.
 
+2020-06-17:
```

Comment:
Please add a blank line for formatting consistency.

Reply:
This has been fixed.